types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
Have you checked the transpiled code and not the source file? This is actually correct import because `TransitionGroup` and `CSSTransition` are converted into CJS modules during compilation.